package.json

```diff
@@ -39,7 +39,7 @@
 		"dev": "run-p --print-label --race 'dev:*'",
 		"dev:backend": "cd backend && make start",
 		"dev:components": "yarn turbo run dev --filter @highlight-run/component-preview...",
-		"dev:frontend": "doppler run -- yarn turbo run dev --concurrency 32 --filter frontend... --filter=!rrweb... --filter=!@highlight-run/ui...",
+		"dev:frontend": "doppler run -- yarn turbo run dev --concurrency 32 --filter frontend... --filter=!rrweb...",
```

Comment:
Kind of unrelated, but brought this back since I realized it was disabled a while back because the Storybook build was broken, but now that's fixed, so I didn't see any harm in building it as part of `dev:frontend`. It's come up as a pain point a few times too in retros 😄